app/Http/Controllers/Api/TaskSearchController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Services\TaskService;
+
+class TaskSearchController extends Controller
+{
+
+    protected $taskService;
+
+    public function __construct(TaskService $taskService)
+    {
+        $this->taskService = $taskService;
+    }
+
+    /**
+     * get all tasks.
+     *
+     * @return mixed
+     */
+
```

Comment:
Your task is to search even though the search has been written already... 
This code you wrote only get all tasks

Reply:
I only did as the title read. is there any available backend tickets that have not been done?